cluster/ci/origin/images/binaries/Dockerfile

```diff
@@ -1,3 +1,15 @@
-FROM source
+FROM build-base
 
-RUN umask 0002 && OS_GOFLAGS="-race" hack/build-go.sh
+ARG PULL_REFS=master
+ARG REPO_OWNER=openshift
+ARG REPO_NAME=origin
+
+ENV UMASK=0002
+
+RUN run.sh release-ci cloneref --src-root=/data
```

Comment:
Are we not sharing this layer now?

Reply:
For the regular (non-test) build I didn't see a use